go.mod

```diff
@@ -388,7 +408,7 @@ require (
 	golang.org/x/term v0.10.0 // indirect
 	golang.org/x/text v0.11.0 // indirect
 	golang.org/x/time v0.3.0 // indirect
-	golang.org/x/tools v0.11.0 // indirect
+	golang.org/x/tools v0.9.3 // indirect
```

Comment:
Why was this downgraded?

Reply:
@ItalyPaleAle interesting.. I don't know.. maybe I had an old branch.. but fixed now